web/package.json

```diff
@@ -98,11 +99,12 @@
     "deploy": "yarn run build && serverless deploy && serverless syncToS3",
     "relay": "relay-compiler --src ./src/ --schema ./data/schema.graphql",
     "relay:watch": "nodemon --watch src/ --watch data/schema.graphql --ignore **/__generated__/**/* --exec \"yarn run relay\"",
-    "test": "npm-run-all -s test:run test:codecov",
+    "test": "npm run test:lint && npm-run-all -s test:run test:codecov",
```

Comment:
For all package.json files, include linting as one of the `npm-run-all` commands. `npm-run-all -s` is just a cross-platform replacement for using `&&` between scripts.